tests/main/lxd/task.yaml

```diff
@@ -70,6 +70,10 @@ execute: |
     # my-nesting-ubuntu nesting container
 
     . /etc/os-release
+    # TODO:2004: remove version check once the image for ubuntu 20.04 is published
+    if [ "$VERSION_ID" == 20.04 ]; then
+        VERSION_ID=19.10
```

Comment:
This may involve copying binaries built in 20.04 into a 19.04 system that may not work. Still, if it happens we should just remove the hack and run this on real 20.04 container or not at all.